q2_feature_table/tests/filter/test_filter_samples.py

```diff
@@ -781,12 +858,12 @@ def test_combine_exclude_ids_and_features_filters(self):
         table = Table(np.array([[0, 1, 3], [0, 1, 2]]),
                       ['O1', 'O2'],
                       ['S1', 'S2', 'S3'])
-        actual = filter_samples(table,
-                                metadata=metadata,
-                                exclude_ids=True,
-                                min_features=1,
-                                max_features=1)
-        self.assertTrue(actual.is_empty())
+        with self.assertRaisesRegex(ValueError, 'table is empty'):
```

Comment:
Same as above comments for comment on line 852/853, and also adding back in the True case for this test.